.github/workflows/pull_request.yaml

```diff
@@ -44,26 +49,43 @@ jobs:
         if: ${{ inputs.enableOpenApiCheck == true }}
         uses: nwestfall/openapi-action@v1.0.2
         with:
-         github_token: ${{ secrets.GITHUB_TOKEN }}
-         file: ${{ inputs.openApiFilePath }}
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          file: ${{ inputs.openApiFilePath }}
 
   security:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@master
-    - name: Run Snyk to check for vulnerabilities
-      uses: snyk/actions/node@master
-      continue-on-error: true
-      env:
-        SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+      - uses: actions/checkout@master
+      - name: Run Snyk to check for vulnerabilities
+        uses: snyk/actions/node@master
+        continue-on-error: true
+        env:
+          SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
 
   tests:
     name: Run Tests
     runs-on: ubuntu-latest
     strategy:
       matrix:
         node: [18.x, 20.x]
-
+    services:
+      # Label used to access the service container
+      postgres:
+        # Docker Hub image
+        image: ${{ (inputs.usePostgres == true) && 'postgis/postgis:13-3.4' || ''}}
```

Comment:
@nitzanmorr have you tested with an empty string ?
Can you please paste a link of a working workflow

Reply:
github will fail to run the action with an empty string because usePostgres is set to be a boolean. If this input is not mentioned in the "with" section, it defaults to false and works as expected.

here's a link of a working workflow with usePostgres on: https://github.com/MapColonies/heartbeat-manager/actions/runs/10285679513

usePostgres not mentioned/false: https://github.com/MapColonies/Task-Liberator/actions/runs/10298441999